cartographer/sensor/compressed_range_data_proto_test.cc

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2016 The Cartographer Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cartographer/sensor/compressed_point_cloud.h"
+#include "cartographer/sensor/point_cloud.h"
+#include "cartographer/sensor/range_data.h"
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "cartographer/common/port.h"
+#include "cartographer/sensor/proto/sensor.pb.h"
+#include "gtest/gtest.h"
+
+namespace cartographer {
+namespace sensor {
+namespace {
+
+TEST(CompressedRangeDataProtoTest, ToProto) {
```

Comment:
If this tests code in `range_data.cc` it should be in the file `range_data_test.cc`. We already test compression there. I'm absolutely not opposed to more tests, but I'd be fine with having a unit test for the proto conversion of CompressedPointCloud for this PR.